Validation/HcalDigis/src/HcalDigisValidation.cc

```diff
@@ -264,7 +264,7 @@ void HcalDigisValidation::booking(DQMStore::IBooker& ib, const std::string bsubd
       book1D(ib, histo, digiAmp);
 
     sprintf(histo, "HcalDigiTask_number_of_amplitudes_above_10fC_%s", sub);
-    book1D(ib, histo, ndigis);
+    book1D(ib, histo, Ndigis);
```

Comment:
WIth this change, ndigis in L181 seems to be unused now, right?

Reply:
Right, indeed...